lib/common/hasher_test.go

```diff
@@ -33,6 +33,12 @@ func TestBlake128(t *testing.T) {
 	require.Equal(t, expected, h)
 }
 
+func TestBlake2b8(t *testing.T) {
+	exp := [8]byte{0xd2, 0xbc, 0x98, 0x97, 0xee, 0xd0, 0x8f, 0x15}
+	res := common.MustBlake2b8([]byte("TaggedTransactionQueue"))
```

Comment:
nit side note, having a `common_test` package in the common directory is just weird AF :smile: